### PR TITLE
KAT Med update

### PR DESCRIPTION
**Pull request by Bob-Murphy**
## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [ ] Enhancement
4. [x] Update

### What have you changed and why?
Information:
Update for [Kat  - Advanced Medical REWRITE](https://steamcommunity.com/sharedfiles/filedetails/?id=2020940806)
- overhauled the detection to not also detect the [deprecated version](https://steamcommunity.com/sharedfiles/filedetails/?id=1626728054) by using an asset that's new
- added new assets

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
- load it
********************************************************
Notes:
[Classnames](https://github.com/Tomcat-SG/KAM/wiki/Classnames)
[KAT - Advanced Medical REWRITE OFFICIAL GUIDE](https://docs.google.com/document/d/1YlY00qoL2gdrbOJJzi5aW19A3HqT7uXflszDYuK7h3M/edit?usp=sharing)

### CODE REVIEW

File: A3A/addons/core/functions/Templates/fn_aceModCompat.sqf [hunk 1]
```diff
@@ -54,16 +54,78 @@ advItems = [
...
+	"kat_bloodIV_B",     //KAT Circulation
+	"kat_bloodIV_B_250",     //KAT Circulation
+	"kat_bloodIV_B_250_N",     //KAT Circulation
+	"kat_bloodIV_B_500",     //KAT Circulation
+	"kat_bloodIV_B_500_N",     //KAT Circulation
+	"kat_bloodIV_B_N",     //KAT Circulation
+	"kat_bloodIV_O",     //KAT Circulation
+	"kat_bloodIV_O_250",     //KAT Circulation
+	"kat_bloodIV_O_250_N",     //KAT Circulation
+	"kat_bloodIV_O_500",     //KAT Circulation
```
Comment (killerswin2) [inline A3A/addons/core/functions/Templates/fn_aceModCompat.sqf:87]: Given that O- blood is generic to most of population most players will grab it. There please choose if you want to keep it in. If you want to keep the O- blood you will need to remove all the other bloods (A, AB, B, O). If you keep the blood remove the blood card as it's not necessary

**Review by killerswin2 — CHANGES_REQUESTED** [review]

**Comment by killerswin2** [comment]
Other than those two remarks this is good.

File: A3A/addons/core/functions/Templates/fn_aceModCompat.sqf (hunk 1 above)
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Templates/fn_aceModCompat.sqf:87]: We can't do that.
A player could have O- and would rely on O- blood.
We should keep everything in so players can decide how they play it.  Like for example if they take only O- or their own blood group.

We could adapt it when we would move not a not-unlimited system and then have a very limited amount of O- blood, but just taking some out makes no sense.

File: A3A/addons/core/functions/Templates/fn_aceModCompat.sqf [hunk 2]
```diff
@@ -54,16 +54,78 @@ advItems = [
...
+	"kat_accuvac",     //KAT Airway
+	"kat_guedel",     //KAT Airway
+	"kat_larynx",     //KAT Airway
+
+	"kat_aatKit",     //KAT Breathing
+	"kat_chestSeal",     //KAT Breathing
+	"kat_Pulseoximeter",     //KAT Breathing
+	"kat_stethoscope",     //KAT Breathing
+
+	"kat_bloodIV_A",     //KAT Circulation
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Templates/fn_aceModCompat.sqf:66]: Would we need to remove the blood the plasma as well as the saline or just the bloods?
I'd say all of them, right?

If so, could you please take over that task of adapting it?

[the remark this answers is not in the document]
Comment (killerswin2) [reply A3A/addons/core/functions/Templates/fn_aceModCompat.sqf:66]: we will only need to do blood. Plasma and saline can stay in. I can do the adapting.

**Review by killerswin2 — APPROVED** [review]
This is good now! Can be merged

**Pull request merged**